homeassistant/components/lawn_mower/const.py

```diff
@@ -0,0 +1,33 @@
+"""Constants for the lawn mower integration."""
+from enum import IntFlag, StrEnum
+
+
+class LawnMowerActivity(StrEnum):
+    """Activity state of lawn mower devices."""
+
+    # Device is in error state, needs assistance
+    ERROR = "error"
```

Comment:
If we write it like this, IDEs will pick it up and show it to the developer

```suggestion
    ERROR = "error"
    """Device is in error state, needs assistance."""
```